src/com/lilithsthrone/game/character/body/Body.java

```diff
@@ -6386,4 +6413,7 @@ public void setTakesAfterMother(boolean takesAfterMother) {
 		this.takesAfterMother = takesAfterMother;
 	}
 
+	public BodyPartTypeInterface randomTypeFrom(BodyPartTypeInterface... values) {
```

Comment:
If you feel inclined to write a doc comment for this, it might be good to mention that simple weighted averages can be achieved by listing a more desireable outcome more than once.

Reply:
I intend to add this after a quick test:
```java
	/**
	 * Returns a randomly chosen BodyPart-type from the list of types (e. g. multiple WingTypes)
	 * provided as parameters. To give one or more types more weight these types can be repeated.
	 * @param values List of BodyPartTypes to choose from randomly
	 * @return The randomly chosen type of the corresponding BodyPart
	 */
 ```